Model/Page.php

```diff
@@ -174,6 +174,11 @@
     protected static $slugifyMethod;
 
     /**
+     * @var int
+     */
+    private $targetRedirectType = 302;
```

Comment:
Do you reproduce the issue in CLI, with the following script:

```php
require './vendor/autoload.php';
new \ReflectionProperty(Sonata\PageBundle\Entity\BasePage::class, 'targetRedirectType');
```

Also, please refrain from posting screenshots of text, it's bad for several reasons I contributed to the sf docs. There is a plain text stack trace at the bottom of the page.